### PR TITLE
Js us37-39

### PR DESCRIPTION
**Pull request by jrsewell400**
Made the following changes:
- Created feature tests for Admin::Merchant controller and index, show and update controller actions.
- Added model test for deactivate_items helper method
- Created view pages for Admin::Merchant index and show
- Updated Merchant table with enabled? attribute, and updated schema
- Added a link to Ship Order to Admin Dashboard
- Added new route and updated "resource" to "resources" under Admin namespace for orders and merchants

### CODE REVIEW

**Review by elguapogordo — APPROVED** [review]
looks solid

File: app/controllers/admin/merchants_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  def index 
+    @merchants = Merchant.all
+  end
+
+  def show 
+    @merchant = Merchant.find(params[:id])
+  end
+
+  def update 
+    merchant = Merchant.find(params[:id])
```
Comment (zachholcomb) [inline app/controllers/admin/merchants_controller.rb:11]: probably want to break this method out into helper methods during refactor phase

File: app/models/merchant.rb [hunk 2]
```diff
@@ -32,4 +32,8 @@ def pending_orders
     orders.distinct.where(status: 0)
   end
 
+  def deactivate_items
+    items.update(active?: false)
```
Comment (zachholcomb) [inline app/models/merchant.rb:36]: so this deactivates all merchants items

File: spec/models/merchant_spec.rb [hunk 3]
```diff
@@ -72,5 +72,14 @@
       expect(@meg.pending_orders).to eq([order_2])
       expect(dog_shop.pending_orders).to eq([order_1])
     end
+
+    it "deactivate_items" do
+      dog_shop = Merchant.create(name: "Brian's Dog Shop", address: '125 Doggo St.', city: 'Denver', state: 'CO', zip: 80210)
+      pull_toy = dog_shop.items.create(name: "Pull Toy", description: "Great pull toy!", price: 10, image: "http://lovencaretoys.com/image/cache/dog/tug-toy-dog-pull-9010_2-800x800.jpg", active?: true, inventory: 32)
+      
+      dog_shop.deactivate_items
```
Comment (zachholcomb) [inline spec/models/merchant_spec.rb:80]: does this work with multiple items in the dog shop?

File: spec/features/admin/merchants/index_spec.rb [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+      @pull_toy = @dog_shop.items.create(name: "Pull Toy", description: "Great pull toy!", price: 10, image: "http://lovencaretoys.com/image/cache/dog/tug-toy-dog-pull-9010_2-800x800.jpg", inventory: 32)
+      @dog_bone = @dog_shop.items.create(name: "Dog Bone", description: "They'll love it!", price: 21, image: "https://img.chewy.com/is/image/catalog/54226_MAIN._AC_SL1500_V1534449573_.jpg", active?:false, inventory: 21)
+      @dog_ball = @dog_shop.items.create(name: "Dog Ball", description: "Awesome dog ball!", price: 5, image: "https://img.chewy.com/is/image/catalog/59155_MAIN._AC_SL1500_V1518033665_.jpg", inventory: 20)
+      @dog_bowl = @dog_shop.items.create(name: "Dog Bowl", description: "Great dog bowl!", price: 7, image: "https://www.talltailsdog.com/pub/media/catalog/product/cache/a0f79b354624f8eb0e90cc12a21406d2/u/n/untitled-6.jpg", inventory: 32)
+    end 
+    
+    it "then I see a 'disable' button next to active mechants that, when clicked, returns me to admin merchant index where I see merchant account disabled and flash message." do 
+      visit admin_merchants_path
```
Comment (steve-meyers) [inline spec/features/admin/merchants/index_spec.rb:20]: Maybe the description for this test should indicate that the items are also getting disabled.

File: spec/models/merchant_spec.rb (hunk 3 above)
Comment (steve-meyers) [reply spec/models/merchant_spec.rb:80]: Yeah, I have the same question.  Deactivating multiple items here would be a better test.

**Review by steve-meyers — COMMENTED** [review]
The only thing I would change is make the model method test for `.deactivate_items` have multiple items being deactivated at once

File: spec/models/merchant_spec.rb (hunk 3 above)
Comment (jrsewell400) [reply spec/models/merchant_spec.rb:80]: I handled this in the feature test, I will add it to the model test as well

File: app/models/merchant.rb (hunk 2 above)
Comment (jrsewell400) [reply app/models/merchant.rb:36]: yep!

File: spec/features/admin/merchants/index_spec.rb (hunk 4 above)
Comment (jrsewell400) [reply spec/features/admin/merchants/index_spec.rb:20]: Will add that extra language to my 40-41 push

**Pull request merged**